.github/actions/setup-builder/action.yaml

```diff
@@ -17,14 +17,13 @@
 
 name: Prepare Rust Builder
 description: 'Prepare Rust Build Environment'
-inputs:
-  rust-version:
-    description: 'version of rust to install (e.g. stable)'
-    required: true
-    default: 'stable'
 runs:
   using: "composite"
   steps:
+    - uses: actions/checkout@v4
```

Comment:
We need to checkout the code in the builder setup to get the `rust-toolchain.toml` file